usb/usb_host_hid/include/usb/hid_host.h

```diff
@@ -16,6 +17,8 @@
 extern "C" {
 #endif
 
+#define HID_STR_DESC_SIZE           32
```

Comment:
The max length of a string descriptor according to USB 2.0 specification is 256-2 bytes. Please add a description as to what this macro actually means.

```suggestion
#define HID_STR_DESC_MAX_SIZE           32
```